targets/targets.json

```diff
@@ -5757,7 +5757,7 @@
         "inherits": ["Target"],
         "default_toolchain": "ARMC6",
         "extra_labels": ["ARM_SSG", "MUSCA_A1"],
-        "forced_reset_timeout": 7,
+        "forced_reset_timeout": 13,
```

Comment:
this should be a separate commit, or even a pull request?

Reply:
@0xc0170 This workaround was done to fix the bug in TF-M bootloader. But it is already fixed in the version of TF-M bootloader imported into Mbed OS. Therefore we don't need this workaround anymore.